File: Walter/LENGUAJE_ESTRUCTURADO_12062023-main/APACHENETBEANS/src/polimorfismo_figurageometrica2d/Circulo.java
package polimorfismo_figurageometrica2d;

public class Circulo extends FiguraGeometrica2D {

    private double radio;

    public Circulo() {
    }

    public Circulo(double radio) {
        this.radio = radio;
    }

    public double getRadio() {
        return radio;
    }

    public void setRadio(double radio) {
        this.radio = radio;
    }

    @Override
    public String toString() {
        return "Circulo{" + "radio=" + radio + '}';
    }

    @Override
    public double area() {
       return Math.PI * Math.pow(this.radio,2);
    }

    @Override
    public double perimetro() {
        return 2 * Math.PI * this.radio;
    }

    @Override
    public String soy() {
        return "Circulo";
    }

}

File: Walter/Python_walter/POO/Ejemplo6/circulo.py
import math
from figurageometrica import FiguraGeometrica

class Circulo(FiguraGeometrica):
      
      def __init__(self, radio=None):
          self.radio = radio

      def perimetro(self):
          return 2 * math.pi * self.radio
      
      def area(self):
          return math.pi * math.pow(self.radio,2)
      
      def soy(self):
          return 'Circulo'